app/assets/javascripts/snowcrash/pages/nodes/notes/table.js.coffee

```diff
@@ -0,0 +1,4 @@
+document.addEventListener "turbolinks:load", ->
+  if $('body.nodes.show').length
+
+    new IndexTable('note')
```

Comment:
Wouldn't this be problematic from a local storage point of view?

If I understand this correctly the storage key will be given by:

```
 +    "project.#{@project}.#{@itemName}_columns" 
```

and if itemName is just "note" the key will be the same for all the nodes in the project?